### PR TITLE
Add getting-started docs

### PR DESCRIPTION
**Pull request by BalbinaK**
It's a first attempt to document what needs to be in place for Warp components to work. 

Section 1. is to some extent based on the corresponding getting-started page of [Warp css-docs](https://warp-ds.github.io/css-docs/getting-started).
Section 2. will probably change when the theme css is applied on a higher level, e.g. based on what domain the application is loaded on.
Section 3. is based on Fabric components docs ([React](https://react.fabric-ds.io/), [Vue](https://vue.fabric-ds.io/) and [Elements](https://elements.fabric-ds.io/)).

getting-started-docs

What's still unclear:
1. There's no mention of the need to apply a css reset for some things to work (like border on Alert), because I wasn't sure how to properly document it. It's the part about our `usePreflight` option on `presetWarp`. 
2. All three Warp component libraries are also available via our Eik CDN server (e.g. https://assets.finn.no/pkg/@warp-ds/react) but it wasn't clear to me if it's something we should document.

Any comments and suggestions are highly appreciated, as always :)

### CODE REVIEW

**Comment by BalbinaK** [comment]
[Add documentation in tech-docs for how to get started using Warp](https://trello.com/c/MevaZg9f/170-add-documentation-in-tech-docs-for-how-to-get-started-using-warp)

File: docs/getting-started/elements.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+## Using Components
+
+Once installed, components can be used in your HTML markup.
+
+```js
+<f-breadcrumbs class="mt-10">
+  <a href="#/url/1">Eiendom</a>
+  <a href="#/url/2">Bolig til salgs</a>
+  <a href="#/url/3" aria-current="page"> Oslo </a>
+</f-breadcrumbs>
```
Comment (magnuh) [inline docs/getting-started/elements.md:24]: Should this be w-breadcrumbs now? 🙂 
...personally I would prefer Norwegian as now, but do we want the examples to be in English? 😄

File: docs/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+## 2. Apply theme
+
+In order for components to apply your application's theme, a respective theme stylesheet should be added to the document. Theme specific stylesheets are vailable via our Eik CDN server:
+* Finn: https://assets.finn.no/pkg/@warp-ds/tokens/v1/finn-no.css
+* Tori: https://assets.finn.no/pkg/@warp-ds/tokens/v1/tori-fi.css
+* Blocket: https://assets.finn.no/pkg/@warp-ds/tokens/v1/blocket-se.css
+
+Add this to your `index.html`:
+
+```js
```
Comment (magnuh) [inline docs/getting-started/index.md:89]: html? 😄

File: docs/getting-started/elements.md (hunk 1 above)
Comment (magnuh) [reply docs/getting-started/elements.md:24]: ...and this code block should be html as well. 😄

File: docs/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+See [UnoCSS CLI](https://unocss.dev/integrations/cli) for more information and examples.
+
+
+### Use with Shadow DOM and Vite
+
+In case of apps using Vite where Warp components are used inside Shadow DOM, the css generated by the UnoCSS plugin needs to be inlined into the Shadow DOM style. See [UnoCSS Vite plugin's shadow-dom mode](https://unocss.dev/integrations/vite#shadow-dom) for more information.
+
+## 2. Apply theme
+
+In order for components to apply your application's theme, a respective theme stylesheet should be added to the document. Theme specific stylesheets are vailable via our Eik CDN server:
```
Comment (AnnaRybkina) [inline docs/getting-started/index.md:82]:
```suggestion
In order for components to apply your application's theme, a respective theme stylesheet should be added to the document. Theme specific stylesheets are available via our Eik CDN server:
```

**Review by AnnaRybkina — APPROVED** [review]
I think it looks amazing! Really great readup with detailed explanations and examples. You've got some good minor comments but I'll approve already 💪  Great work

**Review by Skadefryd — APPROVED** [review]
Very nice :) Tipptopptommelopp!

File: docs/getting-started/elements.md (hunk 1 above)
Comment (BalbinaK) [reply docs/getting-started/elements.md:24]: Good point! Changed to English examples in 01bf8aab15752d40d7fce574788aaf33668ec29d

File: docs/getting-started/index.md (hunk 2 above)
Comment (BalbinaK) [reply docs/getting-started/index.md:89]: 01bf8aab15752d40d7fce574788aaf33668ec29d

File: docs/getting-started/index.md (hunk 3 above)
Comment (BalbinaK) [reply docs/getting-started/index.md:82]: 01bf8aab15752d40d7fce574788aaf33668ec29d

**Pull request merged**